src/controllers/landing.controller.ts

```diff
@@ -2,14 +2,13 @@ import { NextFunction, Request, Response } from "express";
 
 import { logger } from "../utils/logger";
 import * as config from "../config";
-import { deleteApplicationData } from "../utils/application.data";
 
 export const get = (req: Request, res: Response, next: NextFunction) => {
   try {
-    logger.debugRequest(req, `GET LANDING_PAGE`);
+    const landingUrl = config.LANDING_PAGE_URL;
+    logger.debugRequest(req, `GET LANDING_PAGE ${landingUrl}`);
```

Comment:
the logger comment isn't quite correct, the landingUrl is where it will redirect to and not what was submitted. If you wanted to log where we are going , i'd add a separate logger.debugRequest to say 'Redirecting to ${landingUrl}'